### ISSUE TITLE
Ferdi: Searchbar, quote, and some other fields are white as of today

### ISSUE THREAD

**Issue by kyle-ds** [dcferreira/franz-themes#3]
Areas that were not white yesterday but are today:
- Search Bar
- Quotes
- Youtube previews

I'm not great with CSS, but through the inspector I found.

Youtube
```
html[dir] ._3mRez.D4OT6 {
    background-color: var(--incoming-background-deeper);
}
```

Post-load screen 
```
html[dir] ._2zynu {
    background-color: var(--intro-background);
    cursor: default;
    padding-bottom: 28px;
    padding-top: 28px;
}
```

I could search more, but before I do that, is there any advice for fixing? Does WhatsApp normally keep changing these class names?

**Comment by dcferreira** [comment]
Assuming it's the same issue as in https://github.com/vednoc/dark-whatsapp/issues/130, it should be fixed in 612300199c345e6cbd9d193b2372b291d6432565. Let me know if this doesn't fix your problem :)

**Comment by kyle-ds** [comment]
Based on that issue, looks like it would be the same. Thanks.
